Dappeteer 4.0.1 broken dependency

**Describe the bug**
When we use dappeteer as npm package, import from test folder to src will break because dev dependency is not installed (expected). Therefore importing "export const EXAMPLE_WEBSITE = "http://example.org";" to the src folder from test/ will fail due to uninstalled web3 dependency. As a file will be resolved entirely and not just the export part.

**To Reproduce**
install dappeteer v4.0.1 and call method "initSnapEnv()"

**Logs**
```js
Cannot find module 'web3' from '../../node_modules/@chainsafe/dappeteer/dist/test/constant.js'
```

**Expected behavior**
We shouldn't use a test folder when dappeteer is used as a package

**System:**
- OS: Ventura 13.01
- NodeJs version 16
- dAppeteer version 4.0.1
- testing framework jest
- testing framework version 26

--- a/src/constants.ts
+++ b/src/constants.ts
@@ -1,0 +1,1 @@
+export const EXAMPLE_WEBSITE = "http://example.org";

--- a/src/snap/install.ts
+++ b/src/snap/install.ts
@@ -8,7 +8,7 @@ import {
   profileDropdownClick,
 } from "../helpers";
 import { DappeteerPage } from "../page";
-import { EXAMPLE_WEBSITE } from "../../test/constant";
+import { EXAMPLE_WEBSITE } from "../constants";
 import { startSnapServer, toUrl } from "./install-utils";
 import { flaskOnly, isFirstElementAppearsFirst } from "./utils";
 import { InstallSnapResult } from "./types";

